app/models/volunteer.rb

```diff
@@ -70,6 +70,10 @@ class Volunteer < ApplicationRecord
 
   scope :with_hours, (-> { joins(:hours).distinct })
 
+  def billable_hours_sum
+    hours.billable.sum(&:hours) + hours.billable.sum(&:minutes) / 60
+  end
+
```

Comment:
To avoid repeating the logic from `hours_sum` it's also possible to [pass a block to `has_many`](http://api.rubyonrails.org/classes/ActiveRecord/Associations/ClassMethods.html#method-i-has_many) to define methods which will be available on that association, i.e. something like:

```ruby
has_many ... do
  def total_hours
    sum(&:hours) + sum(&:minutes) / 60
  end
end
```

And then you should be able to do `hours.total_hours` and `hours.billable.total_hours` (or with any other scope). I never used this before so not sure this is really how it works :)